client/src/static/mockedDataDemo.ts

```diff
@@ -11,6 +11,8 @@ export const staticFileURLs = [
 
 // eslint-disable-next-line
 export const buildInitialModelsList = ({ SIR_PN, SIR_FN }: any): Model.Model[] => {
+  const PN = GroMEt2Graph.parseGromet(SIR_PN);
+  const FN = GroMEt2Graph.parseGromet(SIR_FN);
```

Comment:
@adamocarolli feel free to chime in here, i believe the pre-parsed gromets are available on minio/d3?

Reply:
They are not, but they can be uploaded by running this script in an offline process and then saving the files to minio. This would avoid the need to do these transforms in the client and would reduce the network cost of transferring the files. 

@YohannParis if you plan to go this route let me know and we can discuss. I think the change would take about an hour or two.

Reply:
To be honest this a refactor of @RosaRomeroGomez implementation.
I'll prefer to move it to Minio, as well as the parsing into the upload of GroMET process.
But I guess, it that for rapid prototyping and while GroMET is still in development it might be overkill. I let Rosa decide.

Reply:
Yes, I know we talked about moving this to minio so we wouldn't need to do the parsing on our side. However, we still don't have the final GroMEts so we can keep doing it this way until everybody has uploaded their final GroMEts to that github repo. Does that sound ok? I want to avoid the scenario where we need to re-run the script many times to get the latest version uploaded in minio. However, I can see the option where we build some pipeline to get those from the github repo, parse them and put it into minio. Up to you guys. I'd leave this PR as it is for now though.

@adamocarolli @mj3cheun @YohannParis